treasury/overview.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Overview
+description: As a Polkadot parachain, Moonbeam will use an on-chain treasury controlled by council members, that allows for stakeholders to submit proposals to further the network.
+---
+
+# Treasury in Moonbeam
+
+![Treasury Moonbeam Banner](/images/treasury/overview-banner.png)
+
+## Introduction
+
+A treasury is an on-chain managed collection of funds. Moonbase Alpha will have a community treasury for supporting network initiatives to further the network. This treasury will be funded by 20% of transaction fees on the network and will be managed by the Council.
+
+## General Definitions
+
+Some important terminology to understand in regards to treasuries:
+
+- **Council** - a group of elected individuals that control how treasury funds will be spent
+- **Proposals** - a plan or suggestion to further the network that is put forth by stakeholders to be approved by the council
+- **Proposal bond** - a deposit equal to a percentage of the total proposal spend amount
+- **Proposal bond minimum** - minimum amount for a proposal bond. This amount must be paid as the bond if it is higher than the deposit percentage
+- **Spend period** - the amount of days, in blocks, during which the treasury funds as many proposals as possible without exceeding the maximum
+- **Maximum approved proposals** - the maximum amount of proposals that can wait in the spending queue
+
+Currently, for Moonbase Alpha:
+
+|             Variable             |     |                                                  Value                                                  |
+| :------------------------------: | :-: | :-----------------------------------------------------------------------------------------------------: |
+|           Proposal bond          |     |                          5% of the proposed spend                                                       |
+|       Proposal bond minimum      |     |                          1 DEV token                                                                    |
+|           Spend period           |     |                          43200 blocks (6 days)                                                          |
+|     Maximum approved proposals   |     |                          100                                                                            |
+
+
+## Community Treasury
+
+To fund the Treasury, 20% of each block's transactions fees will be sent to the Treasury. The remaining 80% of the fees are burned. The Treasury allows stakeholders to submit spending proposals to be reviewed and voted on by the Council. These spending proposals should include initiatives to further the network or boost network engagement. Some network initiatives could include funding integrations or collaborations, community events, network outreach, and more. 
```

Comment:
This first sentence sounds weird, probably "20% of each block's transactions fees will be allocated to it"? Or something similar